js/src/players/content-player.js

```diff
@@ -4,7 +4,10 @@ import HELPERS from '../utils/helpers';
 const CONTENTPLAYER = {};
 
 CONTENTPLAYER.play = function (firstContentPlayerPlayRequest) {
-  if (this.contentPlayer && this.contentPlayer.paused) {
+  if (DEBUG) {
+    FW.log('---- CONTENTPLAYER.play - autoplay - ' + this.autoplay + ', manualPlay - ' + this.manualPlay);
+  }
+  if (this.vastPlayer && this.vastPlayer.paused && (this.autoplay || this.manualPlay)) {
```

Comment:
shouldn't it be this.contentPlayer?

Reply:
Ah, exactly :confused: Thanks for noticing.